sentry-spring-boot-starter/src/main/java/io/sentry/spring/autoconfigure/SentryProperties.java

```diff
@@ -0,0 +1,158 @@
+package io.sentry.spring.autoconfigure;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Configuration for Sentry, example:
+ *
+ * <pre>
+ * sentry:
+ *     enabled: true
```

Comment:
Add `enabled` property to configuration properties, please. I understand, this is not required, but it helps with work with IDE (like IDEA), because IDEA don't understand properties from Conditions, but understands properties from ConfigurationProperties